<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_conf">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Detector Configuration</string>
   </property>
   <widget class="QPushButton" name="button_init">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_powerOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Power On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_powerOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Detector mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_detector_mode">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Window mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_window_mode">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_status">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_acquisition">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Acquisition</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Integration time (s):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_integration_time">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of co-adds:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nb_coadd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of frames:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nb_frames">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total integration time (s):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_integration_time_total">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_frame_nb">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_take_background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Background</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_live">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Live</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_acquire">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Acquire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_halt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Halt</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_visualisation">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>410</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Visualisation</string>
   </property>
   <widget class="QCheckBox" name="checkBox_substract_background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Substract Background</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_avg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Average</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_max">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_min">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ROI8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI8</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_camera">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>621</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
